cmd/influxd/config.go

```diff
@@ -253,11 +252,6 @@ func NewConfig() *Config {
 	c.Broker.MaxTopicSize = DefaultBrokerMaxTopicSize
 	c.Broker.MaxSegmentSize = DefaultBrokerMaxSegmentSize
 
-	// Detect hostname (or set to localhost).
-	if c.Hostname, _ = os.Hostname(); c.Hostname == "" {
-		c.Hostname = "localhost"
-	}
-
```

Comment:
wondering why it wouldn't be good to still have

```go
if c.Hostname == "" {
  c.Hostname = "localhost"
}
```

i'm probably not thinking about something.